tuning/utils/preprocessing_utils.py

```diff
@@ -86,11 +133,24 @@ def get_data_collator(
             Response template to be used for formatting by TRL.
         tokenizer: AutoTokenizer
             Loaded tokenizer object to be used by the collator.
+        max_sequence_length: int
+            Max sequence length to be used for sequence tokenization.
+        training_data_path: str
+            training dataset path
 
     Returns:
         Callable
             Callable collator to be leveraged by the trainer.
     """
+    is_train_data_pretokenized = is_pretokenized_dataset(data_path=training_data_path)
+
+    if is_train_data_pretokenized:
+        return DataCollatorForSeq2Seq(
+            tokenizer=tokenizer,
+            padding=configs.PADDING_STRATEGY_LONGEST,
+            max_length=max_sequence_length,
+            return_tensors="pt",
```

Comment:
nit - I'd suggest not setting `return_tensors="pt"` directly here, since it is the default, and we don't set it when initializing other collators

Reply:
ah, right! updated